nwcsafpps_runner/prepare_nwp.py

```diff
@@ -157,8 +155,10 @@ def update_nwp(starttime, nlengths):
             LOG.info("File: " + str(result_file) + " already there...")
             continue
 
-        dummy, tmp_filename = tempfile.mkstemp(suffix="_" + timestamp + "+" +
-                                               '%.3dH00M' % forecast_step, dir=nwp_outdir)
+        tmp_filename_handle, tmp_filename = tempfile.mkstemp(suffix="_" + timestamp + "+" +
+                                                             '%.3dH00M' % forecast_step, dir=nwp_outdir)
+        os.close(tmp_filename_handle)
+
```

Comment:
you could make a small function instead to return a temporary filename, eg:
```python
def make_temp_filename(*args, **kwargs):
    tmp_filename_handle, tmp_filename = tempfile.mkstemp(*args, **kwargs)
    os.close(tmp_filename_handle)
    return tmp_filename
```

Reply:
Yes, good idea, I will do

Reply:
TLC?

Reply:
Tender Love and Care

Reply:
Function added and used.
Concerning debug-messages, you are right. I have removed a few.